crates/ruff/src/rules/flake8_future_annotations/rules.rs

```diff
@@ -7,20 +7,12 @@ use ruff_python_stdlib::typing::PEP_585_SUBSCRIPT_ELIGIBLE;
 
 use crate::checkers::ast::Checker;
 
-/// ## What it does
-/// Checks for missing `from __future__ import annotations` import if a type used in the
-/// module can be rewritten using PEP 563.
-///
-/// ## Why is this bad?
-/// Pairs well with pyupgrade with the --py37-plus flag or higher, since pyupgrade only
-/// replaces type annotations with the PEP 563 rules if `from __future__ import annotations`
-/// is present.
-///
```

Comment:
This section should not have been removed when updating the docs. I'll update the error message to say something along the lines of "Rule `missing-future-annotations-with-imports` docs are not formatted. The example section should be rewritten to:"

Reply:
See #4312

Reply:
Thanks, fixed! The error message looked copy+pastable but did not contain the full output. Hopefully that can be an autofix via pre-commit